cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtils.java

```diff
@@ -649,6 +650,17 @@ public static Integer reviewId(HttpServletRequest request, boolean twoStepVerifi
     return reviewId;
   }
 
+  /**
+   * Get the execution progress check interval in milliseconds. Default: {@code null}.
+   *
+   * @param request The Http request.
+   * @return Execution progress check interval in milliseconds.
+   */
+  static Long executionProgressCheckIntervalMs(HttpServletRequest request) {
+    String parameterString = caseSensitiveParameterName(request.getParameterMap(), EXECUTION_PROGRESS_CHECK_INTERVAL_MS_PARAM);
+    return parameterString == null ? null : Long.parseLong(request.getParameter(parameterString));
```

Comment:
maybe we can move the sanity check against `MIN_EXECUTION_PROGRESS_CHECK_INTERVAL_MS` here so if fails much faster for operations.

Reply:
Copied the sanity check here for early termination -- kept the original one for other current (e.g. `admin` request), and potential future use-cases of this public method.